packages/apps/dashboard/server/src/common/config/env-config.service.ts

```diff
@@ -10,7 +10,8 @@ const DEFAULT_HMT_PRICE_FROM = 'human-protocol';
 const DEFAULT_HMT_PRICE_TO = 'usd';
 const DEFAULT_HCAPTCHA_STATS_SOURCE =
   'https://foundation-accounts.hmt.ai/support/summary-stats';
-export const HCAPTCHA_STATS_START_DATE = '2022-07-01';
+const DEFAULT_HCAPTCHA_STATS_FILE = 'hcaptchaStats.json';
+export const HCAPTCHA_STATS_START_DATE = '2024-09-09';
```

Comment:
I think we need 2 variables here.
One is for stats from gcp bucket ('2022-07-01'), second for the stats we're going to retrieve via hCaptcha API ('2024-09-14').